src/components/TransactionHistoryRows/TransactionCommon.tsx

```diff
@@ -51,16 +67,16 @@ export const TransactionCommon = ({
           <Row title='status'>
             <Status status={txDetails.tx.status} />
           </Row>
-          <Row title='minerFee'>
-            <Amount
-              value={txDetails.fee?.value ?? '0'}
-              precision={txDetails.fee?.asset.precision ?? 0}
-              symbol={txDetails.fee?.asset.symbol ?? ''}
-            />
-          </Row>
-          <Row title='date'>
-            <TransactionDate blockTime={txDetails.tx.blockTime} />
-          </Row>
+          {minerFeeProps !== undefined && (
+            <Row title='minerFee'>
+              <Amount {...minerFeeProps} />
+            </Row>
+          )}
+          {dateProps !== undefined && (
+            <Row title='date'>
+              <TransactionDate blockTime={txDetails.tx.blockTime} />
+            </Row>
+          )}
```

Comment:
Don't render these if they aren't present - makes it possible to display all rFOX txs without miner fee and date which aren't available from IPFS data. We will add date to IPFS data in separate task.

Reply:
Just noting this affects normal pending transactions, we will no longer render `0` miner fee while the tx is pending. @shapeshift/product I don't see that as necessarily a bad thing, but would like to run it by you.

Reply:
Yah this makes sense to me, we should only show the data if we have the information available.

Reply:
https://github.com/shapeshift/web/pull/7366/commits/73ce76331eb1abc58c937385c7508d511d6d1b53